src/loans/kamino.py

```diff
@@ -189,8 +200,8 @@ def std_health_ratio(self) -> str:
             return 'inf'
         if self.is_zero_deposit:
             return '0'
-        deposited_value = self.collateral_market_value()
-        borrowed_value = self.debt_risk_adjusted_market_value()
+        deposited_value = self.collateral_risk_adjusted_market_value()
+        borrowed_value = self.debt_market_value()
```

Comment:
not sure if its correct. do we need to adjust for borrow factor for loans without special elevation group?

Reply:
That's what I want to test experimentally.